docs/guides/batching.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Batching"
+---
+
+Batching when configured correctly, can significantly reduce strain on high-traffic REST backends. You only need to add a handful of operators to your GraphQL schema (i.e. custom directives) for Tailcall to do most of the heavy lifting for you[^1].
```

Comment:
generally batching is supported for different backend in tailcall. Although, we using REST as an example here, lets use common terms for backend here and probably add that tailcall supports batching for other backend protocol as well at the end of the guide

```suggestion
Batching when configured correctly, can significantly reduce strain on high-traffic backends. You only need to add a handful of operators to your GraphQL schema (i.e. custom directives) for Tailcall to do most of the heavy lifting for you[^1].
```